PhysicsTools/PatAlgos/plugins/PATMuonProducer.cc

```diff
@@ -468,7 +506,57 @@ void PATMuonProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSetu
               } 
           }
       }
-
+      // MC info
+      aMuon.initSimInfo();
+      if (simInfoIsAvailalbe){
+	switch ((*simClassification)[muonBaseRef]){
+	case 0: 
```

Comment:
Is there an enum for these magic numbers?

Reply:
No. There are some conventions I would prefer to avoid breaking them. For pat::Muon there is a simpler enum.